mysql-test/main/mysql_upgrade.test

```diff
@@ -526,6 +526,7 @@ SELECT * FROM mysql.plugin WHERE name='unix_socket';
 --echo #
 
 # The warning appears during mysql_upgrade, before the schema becomes consistent
+call mtr.add_suppression("InnoDB: Fetch of persistent statistics requested for table `mysql`.`gtid_slave_pos` but the required system tables mysql.innodb_table_stats and mysql.innodb_index_stats are not present or have unexpected structure. Using transient stats instead.");
```

Comment:
The regular expression meta-character `.` is not being escaped here, although each occurrence actually is referring to the character literal `.`.